keel-ec2-plugin/src/test/kotlin/com/netflix/spinnaker/keel/ec2/resolvers/NetworkResolverTests.kt

```diff
@@ -44,17 +41,43 @@ import strikt.assertions.hasSize
 import strikt.assertions.isEqualTo
 import java.time.Duration
 
-internal abstract class NetworkResolverTests<T : Locatable<*>> : JUnit5Minutests {
+internal abstract class NetworkResolverTests<T : Locatable<SubnetAwareLocations>> : JUnit5Minutests {
```

Comment:
These tests can be a bit simpler too as they no longer need to deal with resources that use `SimpleLocations`